src/components/LogIn/LogIn.module.css

```diff
@@ -1,11 +1,11 @@
-.userAuth {
+.logIn {
    display: flex;
    gap: 0.5em;
    height: 4em;
    margin-right: 1em;
 }
 
-.authButton {
+.logInButton {
    width: 6em;
```

Comment:
Now that you have only 1 button, we don't need gap (only 1 flex item).
Moreover, we don't need flex at all or to give height for the wrapper, just give ```height: 4em``` for the button itself and then you have less rules and no redundant code, and you give rules directly for the element that needs them.